draft-ietf-tsvwg-careful-resume.xml

```diff
@@ -536,6 +540,20 @@ Observing ...> Connect -> Reconnaissance --------------------> Normal
          (see first_unvalidated_packet_acknowledged in <xref target="sec-QLOG"></xref>).</t>
         </list></t> <!-- End of list of actions -->
 
+        <t>When the flow is controlled using BBR, the careful resume is implemented
```

Comment:
/the careful resume/Careful Resume/ ... to be consistent?

Reply:
Yes.